### ISSUE TITLE
Bower component for angular.easypiechart.js?

### ISSUE THREAD

**Issue by boroth** [rendro/easy-pie-chart#70]
Are you planning on making a bower component for the angular version?

**Comment by chafnan** [comment]
I don't think it is.  There can only be one main section in the `bower.json`.  How do you current publish to npm?  Cause it could be built into the build process to switch the `main` section and publish to npm for both angular and regular.

**Comment by gbelmm** [comment]
+1 please

**Comment by rendro** [comment]
Closing as obsolete — apologies for the long silence.

easy-pie-chart 3.0.0 is a full rewrite. It deliberately drops the AngularJS 1.x directive (AngularJS has been end-of-life since January 2022), Bower and Meteor packaging, the IE7/8 excanvas path, and the never-finished SVG renderer. It also targets modern browsers only.

This report depends on one of those, so there is nothing left to fix rather than the report being wrong. If you still need this, the 3.0 core is framework-agnostic and can be wrapped in a few lines.

https://github.com/rendro/easy-pie-chart/releases/tag/v3.0.0
